src/python/pants/backend/docker/util_rules/docker_binary.py

```diff
@@ -55,7 +55,7 @@ def build_image(
             argv=tuple(args),
             description=(
                 f"Building docker image {tags[0]}"
-                + (f" +{pluralize(len(tags)-1, 'additional tag')}." if len(tags) > 1 else ".")
```

Comment:
This was an unrelated but minor nit change, it wasn't pretty with a ending period after the image name.. 
```
08:57:22.59 [INFO] Completed: Building docker image test-example:1.2.5.
# vs
08:57:22.59 [INFO] Completed: Building docker image test-example:1.2.5
```